### DISCUSSION TITLE
select option does not appear in the repeater

### DISCUSSION THREAD

**Discussion by AlamKM** [filamentphp/filament#8901] [General]
I have problems with the select option inside the repeater, when I make it outside the repeater the query results appear successfully. but when inside the repeater the query results do not appear like the example image I am showing

**Comment by AlamKM** [comment]
`Repeater::make('fpa_details')
                                    ->label('Analysis Results')
                                    ->relationship()
                                    ->schema([
                                        Select::make('parameter_id')
                                            ->label('Parameter')
                                            ->preload()
                                            ->options(function ($get) {
                                                return Parameter::whereHas('itemParameters', function ($query) use ($get) {
                                                    $query->whereHas('relParameter', function ($subQuery) use ($get) {
                                                        $subQuery->where('item_id', $get('../../item_id'));
                                                    });
                                                })->pluck('parameter', 'id');
                                            })
                                            ->searchable()
                                            ->columnSpan([
                                                'sm' => 1,
                                            ]),`

**Reply by leandrocfe** [reply]
Make sure your relationship on the repeater is working
